emmet-mode.el

```diff
@@ -69,7 +69,8 @@
 
 (defconst emmet-mode:version "1.0.10")
 
-(require 'cl-lib)
```

Comment:
Can somebody please revert this changes? emmet-mode produce warning messages again.